e2e/app_test.go

```diff
@@ -66,3 +69,19 @@ func TestAppRetryCount(t *testing.T) {
 		})
 	}
 }
+
+func TestAppDashStat(t *testing.T) {
+	_, s5cmd, cleanup := setup(t)
+	defer cleanup()
+
+	cmd := s5cmd("--stat", "ls")
+	result := icmd.RunCmd(cmd)
+
+	result.Assert(t, icmd.Expected{ExitCode: 0})
+
+	out := result.Stdout()
+
+	assert.Assert(t, strings.Contains(out, "Operation"))
```

Comment:
let's make sure that the output is a valid tsv. (exact match instead of substring match)